pkg/cluster/internal/loadbalancer/config.go

```diff
@@ -37,8 +38,12 @@ global
   log /dev/log local1 notice
   daemon
 
-resolvers docker
+resolvers kinddns
+  {{ if $.Docker -}}
```

Comment:
: /

Reply:
it works for both providers adding both resolvers :shrug: